.github/workflows/int-test-workflow.yml

```diff
@@ -0,0 +1,122 @@
+name: Integration Test WorkFlow
+on:
+  push:
+    branches:
+      - develop
+      - master
+      - feature-appframework
```

Comment:
Just a note: Added for app framework .. preferable to remove while merging back to develop

Reply:
Yes Sirish. I think @pdhanoyasplunk  was exploring if we can add a regex pattern like `feature*`, which covers all the feature branches.

Reply:
@smohan-splunk  We can add a pattern `'feature**` to run against any branch that runs with `feature` keyword.